src/reactive-membrane.ts

```diff
@@ -68,6 +68,24 @@ const defaultValueMutated: ReactiveMembraneMutationCallback = (obj: any, key: Pr
 };
 const defaultValueDistortion: ReactiveMembraneDistortionCallback = (value: any) => value;
 
+export function wrapDescriptor(membrane: ReactiveMembrane, descriptor: PropertyDescriptor, getValue: (membrane: ReactiveMembrane, originalValue: any) => any): PropertyDescriptor {
+    const { set, get } = descriptor;
+    if ('value' in descriptor) {
+        descriptor.value = getValue(membrane, descriptor.value);
+    }
+    if (!isUndefined(get)) {
+        descriptor.get = function() {
+            return getValue(membrane, get.call(this));
```

Comment:
getValue can do the wrapping when needed before returning the value to use-land

Reply:
Can you add a comment here? I had a hard time to understand why get and set are not symmetrical here.